db/migrate/20181031123747_update_default_employment_details.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateDefaultEmploymentDetails < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
You have forgotten to define the Claim class - this will use the one from the app which if you remember, you must never do in a migration

Reply:
Whoops, fixed in 8f81901